lib/manager/tool-versions/extract.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('manager/tool-versions/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('', '.tool-versions', {})).toBeNull();
+    });
+
+    it('extracts dependencies for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentValue: '10.15.0',
+            currentVersion: '10.15.0',
+            datasource: 'npm',
+            depName: 'node',
+          },
+          {
+            currentDigest: 'v1.0.2-a',
+            currentRawValue: 'ref:v1.0.2-a',
```

Comment:
This doesn't look right. Isn't this a github tag or github ref?

Reply:
@rarkins according to their docs, this is "tag/commit/branch to download from github and compile" [ref](http://asdf-vm.com/manage/configuration.html#tool-versions)

Reply:
In that case it needs to be `git-refs`